packages/compat/README.md

```diff
@@ -44,3 +44,15 @@ import { fromVersionedTransaction } from '@solana/compat';
 const legacyVersionedTransaction = getMyLegacyVersionedTransaction();
 const transaction = fromVersionedTransaction(legacyVersionedTransaction);
 ```
+
+### `fromLegacyTransactionInstruction()`
+
+This can be used to convert a legacy `TransactionInstruction` object to a `IInstruction` object.
+
+```ts
+import { fromLegacyTransactionInstruction } from '@solana/compat';
+
+// imagine a function that returns a legacy `TransactionInstruction`
+const legacyInstruction = getMyLegacyInstruction();
+const transaction = fromLegacyTransactionInstruction(legacyVersionedTransaction);
```

Comment:
```suggestion
const instruction = fromLegacyTransactionInstruction(legacyInstruction);
```

Reply:
Ooops, thats what I get for multitasking PRs during zoom calls... thanks for catching!